dom/nodes/Document-createEvent.html

```diff
@@ -99,48 +101,56 @@
   "IDBVersionChangeEvent",
   "InstallEvent",
   "KeyEvent",
-  "MIDIConnectionEvent",
```

Comment:
:!sort in vim uses the locale by default, but I can set LC_ALL=C to do it ASCIIbetically.